### PR TITLE
Add humans.txt file with officers, contributors, and software components

### LINKED ISSUE #36: Implement humans.txt
Give yourselves some credit, and make a `humans.txt` file.

Examples:
`https://www.google.com/humans.txt`
`https://medium.com/humans.txt`
`https://www.python.org/humans.txt`

For more examples, use a search engine with the parameter `inurl:humans.txt`

### PR DESCRIPTION
**Pull request by cyntheticfox**
closes #36

### CODE REVIEW

**Comment by cyntheticfox** [comment]
Non-officer contributors' emails withheld for privacy reasons

**Comment by cyntheticfox** [comment]
I'm thinking about adding the photo link at the bottom of the page, but am not sure on ensuring the order on the page. Should the link be after the social media buttons but before the text, or after the text?

[solid white logo](http://humanstxt.org/img/oficial-logos/humanstxt-isolated-blank.gif)
[solid orange logo](http://humanstxt.org/img/oficial-logos/humanstxt-isolated-orange.gif)
[transparent color logo](http://humanstxt.org/img/oficial-logos/humanstxt-transparent-color.png)
[transparent b/w logo](http://humanstxt.org/img/oficial-logos/humanstxt-transparent-1ink.png)

The image will have to be locally hosted since their site doesn't support https properly

**Comment by lilyinstarlight** [comment]
I would greatly prefer not have the the humans.txt gif at the bottom (I could be convinced to have a link in the footer text though). It's not required by the standard (which really only says the file name and that is must be UTF-8) and is only mentioned in an "If you want" section in the quickstart.

File: _layouts/default.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
         <title>{{ site.name }}{% if page.title %} - {{ page.title }}{% endif %}</title>
         <link rel="shortcut icon" href="/images/shield-small.png">
         <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+        <link rel="author" href="humans.txt">
```
Comment (lilyinstarlight) [inline _layouts/default.html]: Move up one line and change to this (with front `/`):
```html
<link rel="author" href="/humans.txt">
```

File: humans.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	Derek Rodriguez
+	Andrew Samuels
+	Kenneth Simpson
+	Robert Underwood
+	Adam Van Scyoc
+	Kyle Wolfe
+	Noah Wright
+
+
+/* Disclosures */
```
Comment (lilyinstarlight) [inline humans.txt]: Either this should be all caps (which seems to be a convention) or the other headings need to be title-cased. I've no preference.

File: _layouts/default.html [hunk 3]
```diff
@@ -175,6 +176,10 @@ <h2>{% if page.breadcrumbs.heading %}{{ page.breadcrumbs.heading }}{% else %}{{
                     <li><a href="https://www.youtube.com/channel/UCpnGCvDSNgxOVrN34eUlS-Q"><i class="fab fa-youtube fa-2x"></i></a><li>
                 </ul>
                 <p class="copyright">Copyright: <span>2020</span> CU Cyber &lt;cyber@clemson.edu&gt;. Design and Developed by <a href="https://themefisher.com">Themefisher</a></p>
+                <br>
+                <ul class="list-inline text-center">
+                    <li><a href="/humans.txt"><img src="/images/humanstxt.gif"></a></li>
+                </ul>
```
Comment (lilyinstarlight) [inline _layouts/default.html]: As I mentioned in my comment, I would favor not adding this or making it a small link in the `p` tag above.

**Review by lilyinstarlight — CHANGES_REQUESTED** [review]

File: _layouts/default.html (hunk 3 above)
Comment (cyntheticfox) [reply _layouts/default.html]: rolled back commit 9f83f0a

File: _layouts/default.html (hunk 1 above)
Comment (cyntheticfox) [reply _layouts/default.html]: Reordered and fixed (thought I had done that but apparently not) in 566d0c8

File: humans.txt (hunk 2 above)
Comment (cyntheticfox) [reply humans.txt]: Changed to be ALL UPPERCASE in 58f0542

**Review by lilyinstarlight — APPROVED** [review]

**Pull request merged**